node/resolvers/index.ts

```diff
@@ -113,14 +113,15 @@ export const resolvers = {
       ctx.response.status = 200
     },
     setProfile: async (ctx: Context) => {
-      ctx.set('Content-Type', 'application/json')
-      ctx.set('Cache-Control', 'no-cache, no-store')
-      const body: any = await json(ctx.req)
       const {
-        clients: { graphqlServer, checkout, profileSystem },
+        clients: { graphqlServer, checkout, profileSystem, apps },
         vtex: { logger },
       } = ctx
 
+      const settings = await apps.getAppSettings(getAppId()).catch((error) => {
+        logger.error({ message: 'Error getting app settings', error })
```

Comment:
To standardize our logging, please use a `message` like `"setProfile-getAppSettingsError"`. That way we can easier tell what function or handler (in this case `setProfile`) produced the log.